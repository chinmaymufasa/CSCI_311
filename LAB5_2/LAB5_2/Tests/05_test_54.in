2
76
4
2
137
6
2
184
6
2
81
4
2
3
5
-1
