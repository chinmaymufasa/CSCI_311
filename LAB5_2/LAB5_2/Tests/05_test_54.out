76 
137 76 
76 184 137 
137 76 81 184 
3 76 81 137 184 
